GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,13 +8,145 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return season1.count
+        case 1:
+            return season2.count
+        case 2:
+            return season3.count
+        case 3:
+            return season4.count
+        case 4:
+            return season5.count
+        case 5:
+            return season6.count
+        case 6:
+            return season7.count
+        default:
+            return -1
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        
+        switch indexPath.section {
```

Comment:
This function is way too verbose. Instead of manually dequeing all the cells by switching on the section. You could just write an if statement to check whether the section is odd or even and then dequeue the appropriate cell. In your implementation you are making a cell for each section and repeating yourself a lot.